Mail-notification plugin for WP Support Plus

ECarlsson opened the pull request.

Commit:
Added mail-notification plugin for WP Support Plus

Commit:
Fixed quotes

Commit:
Fixed typo

Commit:
Slight tweaks to querying and security.

JohanWinther reviewed (changes requested):
Instead of hard coding the recipient email we can use the site owner email variable.

Commit:
Changed mail-to adress to be WordPress admin mail.

Changed hyperlink url to be dynamic so long as default support pagei s
used. (Could be made entirely dynamic by getting appropiate redirect url
from wpdb, which would require a second query)

Fixed issue with wpdb prefix not being properly parsed in query string.

Slight tweaks.

fysikteknologsektionen-old committed:
Added admin email variable

Commit:
Merge branch 'mail-notification' of https://github.com/ECarlsson/ftek-plugins into mail-notification
